src/.eslintrc.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
             files: ['*.test.js'],
             rules: {
                 'i18next/no-literal-string': 'off',
+                'react/prop-types': 'off',
```

Comment:
Why was this necessary?
And if it was necessary at some place, I would have preferred just some disabling comments there.

Reply:
This applies just to the test files. Reason I added it is because when you define a wrapper component in a test (like I've done in a couple of tests), eslint will complain if you don't add prop-types for that component.

Now I think the rule is good for client-side code, but I don't think it's useful for tests. So instead of adding disable comments for every test, I've just disabled it globally for all tests. I thought that was cleaner.

Reply:
Oops I missed the `files` part.

Reply:
No worries 👍 Thanks for the review!